generate-plugin.js

```diff
@@ -0,0 +1,134 @@
+'use strict'
+
+const {
+  readFile,
+  writeFile,
+  existsSync
+} = require('fs')
+const path = require('path')
+const chalk = require('chalk')
+const generify = require('generify')
+const argv = require('yargs-parser')
+const cliPkg = require('./package')
+const { execSync } = require('child_process')
+const log = require('./log')
+
+const pluginTemplate = {
+  dir: 'plugin',
+  main: 'index.js',
+  types: 'index.d.ts',
+  scripts: {
+    lint: 'standard && npm run lint:typescript',
+    'lint:typescript': 'standard --parser @typescript-eslint/parser --plugin @typescript-eslint/eslint-plugin test/types/*.ts',
+    test: 'npm run lint && npm run unit && npm run typescript',
+    typescript: 'tsd',
```

Comment:
```suggestion
    test:typescript: 'tsd',
```
to follow fastify semantic: https://github.com/fastify/fastify/blob/578602f424314ae9d78cea68cbc2b419b8daec51/package.json#L15